### DISCUSSION TITLE
2020/06/solved-blazor-identityserver-500-30-error-ancm-issue/

### DISCUSSION THREAD

**Discussion by SeanKilleen** [SeanKilleen/seankilleen.github.io#645] [Specific Post Comments]
https://seankilleen.com/2020/06/solved-blazor-identityserver-500-30-error-ancm-issue/

**Comment by SeanKilleen** [comment]
## Historical Comments

2020-07-01, Disqus user [Jim Killeen](https://disqus.com/by/disqus_YWz7szxyRB/):

> I needed to refer to this again today and it worked out perfect! Thanks again for the great write up.

2020-07-01, Me:

> Glad to have the verification from the original source! :-D

**Comment by GFOX91** [comment]
Hi Sean,

Great article, Quick question what are we supposed to do when we have two app services for the same repos, i.e. one for production and one for dev. The certificate I have created for production works fine, however I'm unsure how to handle the dev site, do i point to the same certificate, or generate a new one?

If I generate a new one will I have to manually override the Identity Server options in appsettings.json in the configuration of the app service in azure. 

Thanks,

Greg
